gradio/networking.py

```diff
@@ -36,8 +35,7 @@
 ASSOCIATION_PATH_IN_STATIC = "static/apple-app-site-association"
 ASSOCIATION_PATH_IN_ROOT = "apple-app-site-association"
 
-FLAGGING_DIRECTORY = 'flagged/'
-FLAGGING_FILENAME = 'data.txt'
+FLAGGING_FILENAME = 'flagged.txt'
```

Comment:
Again, this is fine for the default (although I would have probably used something like `log.txt`), but consider allowing user to override

Reply:
Changed it to `log.txt`, but haven't added it as a parameter. We're starting to have too many parameters in `Interface` right now. Do you think that's an annoyance?